misc/dbt-materialize/dbt/include/materialize/macros/tests/helpers.sql

```diff
@@ -0,0 +1,9 @@
+{% macro cluster_test_sql(main_sql, fail_calc, warn_if, error_if, limit, cluster) -%}
```

Comment:
Was expecting to name this `materialize__ get_test_sql` but getting a max recursion error since it seemed to be referencing itself